README.md

```diff
@@ -41,6 +41,28 @@ Spanned styledHtml = flavoredHtml.parse(htmlString);
 
 Keep in mind that if you don't set anyting, your spannable will have **no styling**, which means all your text will simply have the HTML tags stripped down!
 
+## Loading images
+
+You'll have to implement either `ImgLoader` to load images, if you need to load bitmaps (e.g. from the network) you can use `ImgLoader.BitmapLoader`:
```

Comment:
https://en.wiktionary.org/wiki/e.g.#Usage_notes

> "E.g." and its examples are typically set off from the rest of the sentence by punctuation. In US English, "e.g." is typically followed by a comma, while the comma is often omitted in British English.

It depends apparently, but since I'm Roman and in latin the comma was there, I'll add it :speak_no_evil: